### ISSUE TITLE
Your code sample (in the text) is still wrong!

### ISSUE THREAD

**Issue by MarqueIV** [dotnet/docs#12502]
You state "A TemplateBinding is an optimized form of a Binding for template scenarios, analogous to a Binding constructed with {Binding RelativeSource={RelativeSource TemplatedParent}}" then follow it with "A TemplateBinding is always a one-way binding, even if properties involved default to two-way binding."  I was the original person to call out the one-way issue that got the 2nd part added, explicitly saying it's always one-way, but your code should *also* be updated from this... 

    {Binding RelativeSource={RelativeSource TemplatedParent}}

to this

    {Binding RelativeSource={RelativeSource TemplatedParent}, Mode=OneWay}

because again, the first statement is *not* true without the mode.  Please update.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dfea2d51-7634-0bf3-d3f2-0dbcff0467d8
* Version Independent ID: 823dcfc2-6c2b-debc-ce1a-882e5bd1d9bc
* Content: [TemplateBinding Markup Extension](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/templatebinding-markup-extension#feedback)
* Content Source: [docs/framework/wpf/advanced/templatebinding-markup-extension.md](https://github.com/dotnet/docs/blob/master/docs/framework/wpf/advanced/templatebinding-markup-extension.md)
* Product: **dotnet-framework**
* Technology: **dotnet-wpf**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by Thraka** [comment]
Thank you, I'll update this.

**Comment by MarqueIV** [comment]
Glad to hear that. I’m just looking out for other developers. This was something that derailed me literally for a day or two trying to figure out why something wasn’t working when according to the documentation it should have. It wasn’t until digging around through the source code that we figured out the documentation was incorrect. There’s even a question up on stack overflow about this. That’s when I reported it to you back then, and the documentation was half updated. Glad we can finally close the loop here.  Really trying to help spare other developers the trouble that I went through.
